### PR TITLE
Add web components supported

### PR DESCRIPTION
**Pull request by VandeurenGlenn**
Not sure if this is the appropiate approach, what do you think?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,6 +109,23 @@ $2Behavior = [$2BehaviorImpl];
 <script src="${0:bower_components}/webcomponentsjs/webcomponents-lite.js"></script>
 ```
 
+### [wcs] load web components polyfill when not supported
+
+```html
+<!-- Load Web Components when not supported -->
+<script>
```
Comment (robdodson) [inline README.md]: I'd remove the script tags. I usually do this in a separate .js file

File: README.md [hunk 2]
```diff
@@ -183,6 +203,49 @@ var ${2:Widget} = document.registerElement('${3:my-widget}', {
...
+  <meta name="theme-color" content="black">
+  <link rel="icon" href="icon.png">
+
+  <!-- Safari / iOS -->
+  <meta name="apple-mobile-web-app-capable" content="yes">
+  <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <link rel="apple-touch-icon-precomposed" href="apple-touch-icon.png">
+
+  <!-- Check if Web Components are supported-->
+  <script>
```
Comment (robdodson) [inline README.md:229]: I think this is ok. I usually do this in a separate JS file but that's not a requirement. It's good to give it to folks in an easy to use format like this as well.

File: README.md (hunk 1 above)
Comment (VandeurenGlenn) [inline README.md]: Ok, I should have known this :smile: [Building Progressive Web Apps with Polymer](https://youtu.be/g7f1Az5fxgU?t=9m46s)

File: README.md (hunk 2 above)
Comment (VandeurenGlenn) [inline README.md:229]: Or add some more info in the comment? 

`<!-- Check if Web Components are supported, you can also do this in a separate javascript file -->`

Comment (robdodson) [inline README.md:229]: Nah I think it's fine without the comment. I figure folks will assume they can move it to a separate script

**Comment by robdodson** [comment]
Thanks for this!

**Pull request merged**